### ISSUE TITLE
I wanted to use MicroROS without any custom transport or RMW. Is it possible to bypass the RMW and use only the publish-subscribe?

### ISSUE THREAD

**Issue by NagaHimanshu** [micro-ROS/rmw_microxrcedds#266]

**Comment by pablogs9** [comment]
Please elaborate on your issue, requirements and goals.

**Comment by NagaHimanshu** [comment]
I wanted to develop a basic publish-subscribe model using MicroROS without any UART/TCP/UDP. I'd like to open two threads using FreeRTOS, one thread for publish and another thread for subscribe. For this I created a dummy transport which just fills up memory. I attached a piece of code below. So, is it possible to having this kind of scenario?

[New Text Document.txt](https://github.com/micro-ROS/rmw_microxrcedds/files/9424006/New.Text.Document.txt)
